roles/openstack-package/tasks/yum.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "install {{ project_name }} package"
+  yum:
+    name: openstack-{{ project_name }}
+    state: present
+  register: project_package
+  until: project_package|succeeded
+  retries: 5
+
+- name: ensure project config directory exists
+  file: name="/etc/{{ project_name }}" state=directory
+  when: openstack_package.rootwrap|bool
+
+- name: stat rootwrap.d
```

Comment:
I don't think we need to check for this existence before removing it.
The file > state=absent action will succeed if the dir is there or not